cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -168,6 +170,31 @@ public void query(HttpRequest request, HttpResponder responder,
     }
   }
 
+  @POST
```

Comment:
Shouldn't this be a ``DELETE``? Also, perhaps, we should send out an API review?